services/clusters-keeper/Makefile

```diff
@@ -3,3 +3,25 @@
 #
 include ../../scripts/common.Makefile
 include ../../scripts/common-service.Makefile
+
+.env: .env-devel ## creates .env file from defaults in .env-devel
+	$(if $(wildcard $@), \
+	@echo "WARNING #####  $< is newer than $@ ####"; diff -uN $@ $<; false;,\
+	@echo "WARNING ##### $@ does not exist, cloning $< as $@ ############"; cp $< $@)
+
+
+show-settings: .env ## shows local test settings (running bare metal against AWS)
+	# setting up dependencies
+	@docker compose up --detach
```

Comment:
deploying "dependencies" to `show-settings` sounds to be unintuitive? Why would you need that?

Reply:
well I'll just remove it